tests/text.test.js

```diff
@@ -0,0 +1,22 @@
+const fs = require("fs");
+const md5 = require('md5');
+
+
+test('check that the pages.jsonl file exists in the collection under the pages folder', () => {
+  expect(fs.existsSync('crawls/collections/wr-net/pages/pages.jsonl')).toBe(true);
+});
+
+test('check that the pages.jsonl file exists in the wacz under the pages folder', () => {
+  expect(fs.existsSync('crawls/collections/wr-net/wacz/pages/pages.jsonl')).toBe(true);
+});
+
+test('check that the hash in the pages folder and in the unzipped wacz folders match', () => {
+  const browsertrix_hash = md5(JSON.parse(fs.readFileSync('crawls/collections/wr-net/wacz/pages/pages.jsonl', 'utf8').split('\n')[1])['text']);
```

Comment:
also very minor, could rename this to `crawl_hash` or `collection_hash`, at first i was confused what it meant and why it was different from wacz_hash in the first place : )